website/www/site/content/en/get-started/downloads.md

```diff
@@ -88,6 +88,14 @@ versions denoted `0.x.y`.
 
 ## Releases
 
+### 2.42.0 (2022-10-04)
+Official [source code download](https://www.apache.org/dyn/closer.cgi/beam/2.42.0/apache-beam-2.42.0-source-release.zip).
```

Comment:
nit: you may need to change line 100 to match 108

Reply:
Good catch. Fixed